main.tf

```diff
@@ -143,3 +143,47 @@ module "existing_key_ring_keys" {
   rotation_interval_month  = each.value.rotation_interval_month
   dual_auth_delete_enabled = each.value.dual_auth_delete_enabled
 }
+
+##############################################################################
+# Context Based Restrictions
+##############################################################################
+
+locals {
+  default_operations = [{
+    api_types = [
+      {
+        "api_type_id" : "crn:v1:bluemix:public:context-based-restrictions::::api-type:"
+      },
+      {
+        "api_type_id" : "crn:v1:bluemix:public:context-based-restrictions::::platform-api-type:"
+      }
+    ]
+  }]
+}
+
+module "cbr_rule" {
```

Comment:
We should add a validation to check that the CBR input is only provided whenever the KMS instance in use is Key Protect so it will error at plan time if HPCS is in use to avoid unintended behavior of applying CBR rules that are not applicable to the instance